### ISSUE TITLE
How to disable Dynamic Island?

### ISSUE THREAD

**Issue by hi-im-karma** [qnblackcat/uYouPlus#1309]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I have read the FAQ

### Device information

- iOS version: 17.3.1
- Device model: iPhone 14 pm
- Sideload tool (AltStore, Sideloadly, TrollStore, etc.): MapleSigner

### Describe the issue

For the first time since switching to iPhone I installed an app outside the AppStore and it’s uyouplus. Now I’ve been annoyed by the fact that it’s showing me the Dynamic Island which wasn’t happening with stock YT. 

How can I disable it? It’s really annoying, as if it thinks I’m playing some audio app.
![IMG_1634](https://github.com/qnblackcat/uYouPlus/assets/73917917/8a6603d8-a30c-455c-913a-bf4cc75d6eb1)

### Steps to reproduce

1. Play any video
2. Dynamic Island will show…

### Are you using the latest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus

### Does the issue happen with the official YouTube app from the App Store?

❌ No, I cannot reproduce this issue in the official YouTube app from the App Store

**Comment by therealFoxster** [comment]
Have you tried rebooting your device? I've never seen this happen before. Otherwise you can try uninstalling and reinstalling the app see if it helps. Let me know how it goes.

**Comment by hi-im-karma** [comment]
> Have you tried rebooting your device? I've never seen this happen before. Otherwise you can try uninstalling and reinstalling the app see if it helps. Let me know how it goes.

Yes I did reboot my phone. It seems to be an actual issue as others are having it aswell: https://github.com/qnblackcat/uYouPlus/issues/658

It seriously bothers me. I purchased 1 year MapleSign PRO and installed uyou+ thru their app.

**Comment by qnblackcat** [comment]
I know this sounds dumb, but could you sideload the IPA via AltStore (free acc) and let me know if the issue persits?

**Comment by hi-im-karma** [comment]
I don't have a PC/Mac that I could use it, and honestly don't want to deal with AltStore. I used MapleSign PRO to install it. It's an issue on both uyou+ and uyouenhanced cuz they're not optimized for iPhone 14 Pro (Max) models. 
I read it's because the apps are spoofing YouTube Premium, which conflicts with the Dynamic Island, and there's no way to disable this spoofing. If this is really the case. 

But I already pay for YouTube Premium, and disabled all background playback and the "No ads" tweak cuz I don't need it. I don't have ads by default. 
I only use YT mods because of Sponsorblock and tons of additional features.

**Comment by GrPK** [comment]
> I know this sounds dumb, but could you sideload the IPA via AltStore (free acc) and let me know if the issue persits?

Hey, I tried this way and it works as expected. But what's the problem with using a signing service? I use Signoulous and the problem with Dynamic Island is present here

**Comment by IceBlueLugia** [comment]
Is there any chance of this being fixed? I've been using Signulous and it's had this issue ever since I upgraded to the 14PM. Also just odd that clicking the video title in the Lock Screen, Dynamic Island, and control center don't do anything. SideStore works fine but it doesn't give me notifications :(
